content/events/2020-03-13-meetup.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: event
 image: '/img/meetup-header-with-logo.jpg'
-title: "March 13, 2020, NL-RSE Meetup"
+title: "NL-RSE Meetup"
```

Comment:
I removed this date because it is already shown in the agenda widget. Can someone check if this is problematic? (Perhaps we mess with the RSS feed by changing past posts?)

Reply:
@c-martinez what do you think?

Reply:
Yes, I think putting the topic as part of the title would look nicer.